### PR TITLE
prevent kyt to hang if webpack config run fails

### PR DESCRIPTION
**Pull request by thiagopnts**
the try-catch  gets only config validation errors, if something goes wrong after that, e.g. EACCESS, etc kyt will hang waiting for the `done` hook that will never be called

### CODE REVIEW

File: packages/kyt-core/utils/webpackCompiler.js [hunk 1]
```diff
@@ -7,7 +7,12 @@ module.exports = (webpackConfig, cb) => {
 
   // Compile the webpack config
   try {
-    webpackCompiler = webpack(webpackConfig);
+    webpackCompiler = webpack(webpackConfig, err => {
+      if (err) {
+        logger.error(`${type} webpack error:\n`, err);
+        process.exit();
```
Comment (tizmagik) [inline packages/kyt-core/utils/webpackCompiler.js]: Should this be an exit code of 1?

Comment (delambo) [reply packages/kyt-core/utils/webpackCompiler.js]: Yes good call!

**Comment by delambo** [comment]
As you can see in the starter kyt end to end test, something weird is going on when starting up the dev server. I'll try to debug some more later.

Note on how to reproduce:
```
npm run clean-bootstrap
cd packages/kyt-starter-universal/starter-src
node_modules/.bin/kyt dev
# Should yield:  ❌ Server build failed
```

**Comment by delambo** [comment]
I couldn't get this to work, per above. I'm going to close for now. If anyone wants to give it a try, please reopen.

**Pull request closed without merging**